src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -44,6 +44,8 @@
   private static final int workEndHour = 18;
   private static final int personalEndHour = 23;
   private static final int numDays = 5;
+  private static final String summary = "Read emails";
+  private static final String calendarId = "primary";
```

Comment:
Nit: PRIMARY_CALENDAR_ID

Reply:
Changed it to a variable that gets the parameter from js.